app/controllers/form_controller.rb

```diff
@@ -102,6 +103,12 @@ def responses_for_page(page)
         result[question.id] = question_params
       end
 
+      if %w[checkbox].include?(question.type) &&
```

Comment:
Some context for why this is here - there's precedent back in the old system for a distinct value (`pregblank`) to represent empty answers to this question.

Reply:
Currently looking into whether we can replace this with the divider system used elsewhere.